### ISSUE TITLE
Inception V4 预训练模型加载问题

### ISSUE THREAD

**Issue by JIANZM** [zhang0jhon/AttentionOCR#54]
代码提示"""
[0121 15:54:33 @sessinit.py:87] WRN The following variables are in the graph, but not found in the checkpoint: InceptionV4/attention_lstm/word_embedding/W_wemb, InceptionV4/attention_lstm/feature_map_attention/init_mean/W_init_c, InceptionV4/attention_lstm/feature_map_attention/init_mean/W_init_h, InceptionV4/attention_lstm/feature_map_attention/attention_x/W, InceptionV4/attention_lstm/feature_map_attention/attention_h/W_h, InceptionV4/attention_lstm/feature_map_attention/att/W_att, InceptionV4/attention_lstm/feature_map_attention/att/b_att, InceptionV4/attention_lstm/softmax/softmax_w, InceptionV4/attention_lstm/softmax/softmax_b, InceptionV4/attention_lstm/attention_to_embedding/W_attention_wemb, InceptionV4/attention_lstm/attention_to_embedding/W_hidden_wemd, InceptionV4/attention_lstm/lstm_cell/lstm_W, InceptionV4/attention_lstm/lstm_cell/lstm_U, InceptionV4/attention_lstm/lstm_cell/lstm_Z, InceptionV4/attention_lstm/lstm_cell/lstm_b, learning_rate
"""
有很多变量在InceptionV4官方checkpoint文件都没找到，请问这种情况正常吗
